fyle_accounting_mappings/models.py

```diff
@@ -543,7 +543,7 @@ class Meta:
 
     @staticmethod
     def create_or_update_mapping(source_type: str, destination_type: str,
-                                 source_value: str, destination_value: str, destination_id: str, workspace_id: int):
+                                 source_value: str, destination_value: str, destination_id: str, workspace_id: int, app_name: str):
```

Comment:
`app_name: str = None` - we might use this method in apis directly, don't wanna risk this addition